site2/docs/txn-use.md

```diff
@@ -20,21 +20,29 @@ This section provides an example of how to use the transaction API to send and r
 
 2. Enable transaction. 
 
-    Change the configuration in the `broker.conf` file.
+    Change the configuration in the `broker.conf` or `standalone.conf` file.
 
     ```
+    //mandatory configuration, used to enable transaction coordinator
     transactionCoordinatorEnabled=true
+   
+    //mandtory configuration, used to create systemTopic used for transaction buffer snapshot
     systemTopicEnabled=true
+   
     ```
 
-    If you want to enable batch messages in transactions, follow the steps below.
+    * If you want to enable batch messages in transactions, follow the steps below.
```

Comment:
If you want to ack batch message with transaction should enable acknowledgmentAtBatchIndexLevelEnabled, follow the steps below.